src/DynamoPackages/PackageLoader.cs

```diff
@@ -76,9 +76,20 @@ public string DefaultPackagesDirectory
 
         private readonly List<string> packagesDirectoriesToVerifyCertificates = new List<string>();
 
+        // The standard library directory is located in the same directory as the DynamoPackages.dll
+        internal string StandardLibraryDirectory =>
+            Path.Combine(Path.GetDirectoryName(Assembly.GetAssembly(GetType()).Location),
+                @"Standard Library", @"Packages");
+
         public PackageLoader(string overridePackageDirectory)
-            : this(new[] { overridePackageDirectory })
```

Comment:
Why can't we continue to call this constructor? That way the new directory needs to be added to only that constructor.

Reply:
Thanks, I totally misread this code.